apps/gallery/test/marionette/apps/fakeactivitycaller/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Fake Gallery Activity Caller</title>
```

Comment:
Updated App name and respective variable names to refer  'Gallery Activity Tester'